src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    public String getRandomBall() {
+        setColor(getRandomColor());
+        setNumber(new Random().nextInt(100));
+        return "color is: " + getColor() + ", number is: " + getNumber();
+    }
```

Comment:
at the moment you're returning String, but this method must return Ball with random values.